LayoutTests/svg/animations/animate-zero-keyPoints-should-not-crash.html

```diff
@@ -0,0 +1,3 @@
+<svg zoomAndPan="magnify">
```

Comment:
It it is good to have a well-formed layout test. I would suggest using something like this:

```
<body>
    <svg>
        <text>
            Passes if it does not crash.
            <animateMotion path="m 1,0" keyPoints="0" keyTimes="0" restart="always"/>
        </text>
    </svg>
    <script>
        if (window.testRunner)
            testRunner.dumpAsText();
    </script>
</body>
```
There is no need to add the attribute `zoomAndPan="magnify"`. I think the bug can be reproduced without it.

Reply:
Oh, duh! I'll double-check that the `zoomAndPan` thing isn't needed and update the test.